docs/reference_wgpu.rst

```diff
@@ -52,6 +52,20 @@ to use your own version of that library instead, set the ``WGPU_LIB_PATH``
 environment variable.
 
 
+How this API differs from WebGPU
+--------------------------------
+
+This API is derived from the WebGPU spec, but differs in a few places:
+
+
+.. autodata:: wgpu.base.apidiff
+    :annotation: Differences of base API:
+
+
+.. autodata:: wgpu.backends.rs.apidiff
+    :annotation: Differences of rs backend:
+
```

Comment:
I'm really happy about this part. The codegen enforces exactly the WebGPU API, except where methods are decorated. And all decorations are collected and end up here in the docs!